test/pokemon.output.ts

```diff
@@ -51,9 +52,10 @@ describe('Pokemon Output', () => {
             item => expect(typeof item.camera).to.equal('object', 'camera type'),
             item => expect(item.camera.cylinderRadius).to.not.equal(undefined, 'camera.cylinderRadius'),
             item => expect(item.camera.diskRadius).to.not.equal(undefined, 'camera.diskRadius'),
-            item => expect(item.camera.shoulderModeScale).to.not.equal(undefined, 'camera.shoulderModeScale'),
-
-
+            item => expect(item.name === 'Caterpie' || item.camera.shoulderModeScale !== undefined).to.equal(true, 'camera.shoulderModeScale'),
```

Comment:
Maybe we should move specific Pokemon tests to a new test-block. But the tests are fine!